### PR TITLE
cli: Add prompt when Ctrl-C pressed

### PR DESCRIPTION
**Pull request by hwware**
When I play the pubsub command in console, I am confused by the following scenario:

image

The reason is that when I press Ctrl-C, client exits current connection and reconnects to the server.
Thus I add one prompt message to make everyone clear what it happens.

image

### CODE REVIEW

File: src/valkey-cli.c [hunk 1]
```diff
@@ -2273,6 +2273,8 @@ static void cliWaitForMessagesOrStdin(void) {
             /* Ctrl-C pressed */
             config.blocking_state_aborted = 0;
             config.pubsub_mode = 0;
+            printf("Exiting current connection, ready to reconnect to Valkey server... \n");
```
Comment (zuiderkwast) [inline src/valkey-cli.c]:
```suggestion
            printf("Closing current connection. Ready to reconnect to Valkey server... \n");
```

**Review by zuiderkwast — APPROVED** [review]
Good. Just a minor wording and grammar suggestion.

File: src/valkey-cli.c (hunk 1 above)
Comment (enjoy-binbin) [reply src/valkey-cli.c]: Receive a ctrl C, re-establish the connection?

i am not an expert on it, both are ok.

**Review by enjoy-binbin — APPROVED** [review]

File: src/valkey-cli.c (hunk 1 above)
Comment (zuiderkwast) [reply src/valkey-cli.c]: I think the user knows he pressed Ctrl-C. What he doesn't know is that valkey-cli closes the connection and opens a new one.

Re-establish, reconnect, I don't know. Same same.

**Review by zuiderkwast — APPROVED** [review]
Approved again. :grin:

**Comment by hwware** [comment]
Thanks Merge it before holiday ^_^

**Pull request merged**